File: Examples/VBNetExamples/PDFXEditSimple/frmPassword.vb
Public Class frmPassword
    Private m_password As String

    Public Function GetPassword() As String
        Return m_password
    End Function

    Private Sub btnOK_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnOK.Click
        m_password = txtPassword.Text
        Close()
    End Sub

    Private Sub btnCancel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnCancel.Click
        m_password = ""
        Close()
    End Sub
End Class

File: Examples/C#Examples/PDFXEditSimple/frmPassword.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace PDFXEditSimple
{
    public partial class frmPassword : Form
    {
        string m_password;

        public string GetPassword()
        {
            return m_password;
        }
               
        public frmPassword()
        {
            InitializeComponent();
        }

        private void btnOK_Click(object sender, EventArgs e)
        {
            m_password = txtPassword.Text;
            Close();
        }

        private void btnCancel_Click(object sender, EventArgs e)
        {
            m_password = "";
            Close();
        }
    }
}